github/spec/features/validate_repos_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "../spec_helper"
+require_relative "../../lib/validate_repos"
+
+RSpec.describe ValidateRepos do
+
+  before do
+    @repo_mock = double("Repo", topics: ["govuk"], archived: false)
+    allow(@repo_mock).to receive(:[]).with("name").and_return("this-is-a-govuk-repo")
+    allow(@repo_mock).to receive(:full_name).and_return("this-is-a-govuk-repo")
+    allow(@repo_mock).to receive(:[]).with(:full_name).and_return("this-is-a-govuk-repo")
+    @octokit_mock = double("Octokit::Client", org_repos: [@repo_mock])
+  end
+
+  it "should ignore any repos that exist in repos.json AND are tagged govuk in GitHub" do
+    repos = [{
+      "app_name": "this-is-a-govuk-repo",
+    }]
+
+    stub_repos_json(repos)
+
+    expect { ValidateRepos.new(@octokit_mock).verify_repo_tags }.to output("\e[0;32;40mUntagged govuk repos: No mismatches found.\e[0m\n\n\e[0;32;40mFalsely tagged govuk repos: No mismatches found.\e[0m\n").to_stdout
+  end
+
+  it "should alert if finds an untagged repo in repos.json" do
+    repos = [
+      { "app_name": "this-is-a-govuk-repo", },
+      { "app_name": "this-govuk-repo-is-not-tagged!", }
+  ]
```

Comment:
```suggestion
    ]
```

Reply:
You have the eyes of an owl, didn't spot this off indentation, thank you, fixed.